### DISCUSSION TITLE
"smooth" scrolling capability

### DISCUSSION THREAD

**Discussion by erlendnagel** [paulscottrobson/neo6502-firmware#266] [Ideas]
I didn't find it, so maybe it doesn't exist yet, or I've misunderstood the concept, but I think in order to create games that scroll( horizontally or vertically), it would be really helpful to be able to determine which pixel row/column is the first (and last) to be drawn. If it is possible to set that upto the width/height of a tile, smoot scrolling can be implemented.

**Comment by paulscottrobson** [comment]
I'm not quite sure what you mean ?

The simplest way to do scrolling is to have a tilemap window and adjust the scrolling offset ; if you want a really long or endless scrolling area then this tilemap would have to be gamed. A good example of this is "Frogger" which is actually 10 one tile high tilemaps all with different scrolling.
